packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -146,7 +153,7 @@ class Braze {
           .getUser()
           .setDateOfBirth(date.getUTCFullYear(), date.getUTCMonth() + 1, date.getUTCDate());
       } catch (error) {
-        logger.error('Error in setting birthday', error);
+        logger.error(`Error in setting birthday - ${JSON.stringify(error)}`);
```

Comment:
you probably want to use the utility method to stringify without circular here

Reply:
updated